frontend/ee/components/UsersPage/UsersTable.jsx

```diff
@@ -0,0 +1,180 @@
+        >
+          <table data-testid="usersTable" className="table table-vcenter" disabled={true} style={{ height: '100%' }}>
+            <thead>
+              <tr>
+                <th data-cy="name-title">Name</th>
+                <th data-cy="email-title">Email</th>
+                {users && users[0]?.status ? <th data-cy="status-title">Status</th> : <th className="w-1"></th>}
+                <th className="w-1"></th>
+              </tr>
+            </thead>
+            {isLoading ? (
+              <tbody className="w-100" style={{ minHeight: '300px' }}>
+                {Array.from(Array(4)).map((_item, index) => (
+                  <tr key={index}>
+                    <td className="col-2 p-3">
+                      <div className="d-flex align-items-center">
+                        <Skeleton circle="15%" className="col-auto" style={{ width: '35px', height: '35px' }} />
+                        <Skeleton className="mx-3" width={100} />
+                      </div>
+                    </td>
+                    <td className="col-4 p-3">
+                      <Skeleton />
+                    </td>
+                    {users && users[0]?.status && (
+                      <td className="col-2 p-3">
+                        <Skeleton />
+                      </td>
+                    )}
+                    <td className="text-muted col-auto col-1 pt-3">
+                      <Skeleton />
+                    </td>
+                    <td className="text-muted col-auto col-1 pt-3">
+                      <Skeleton />
+                    </td>
+                  </tr>
+                ))}
+              </tbody>
+            ) : (
+              <tbody>
+                {users.map((user) => (
+                  <tr key={user.id}>
+                    <td>
+                      <Avatar
+                        avatarId={user.avatar_id}
+                        text={`${user.first_name ? user.first_name[0] : ''}${user.last_name ? user.last_name[0] : ''}`}
+                      />
+                      <span
+                        className="mx-3"
+                        style={{
+                          display: 'inline-flex',
+                          marginBottom: '7px',
+                        }}
+                        data-cy="user-name"
+                      >
+                        {user.name}
+                      </span>
+                    </td>
+                    <td className="text-muted">
+                      <a className="text-reset user-email" data-cy="user-email">
+                        {user.email}
+                      </a>
+                    </td>
+                    {user.status && (
+                      <td className="text-muted">
+                        <span
+                          className={`badge bg-${
+                            user.status === 'invited' ? 'warning' : user.status === 'archived' ? 'danger' : 'success'
+                          } me-1 m-1`}
```

Comment:
Let's simplify this using cx package
```suggestion
      import cx from 'classnames'
      ...
      className={cx('badge me-1 m-1', { 
        'bg-warning': user.status === 'invited', 
        'bg-danger': user.status === 'archived',
        'bg-success': user.status === 'active',
     })}
```